tests/pytests/test_mints.py

```diff
@@ -43,3 +43,56 @@ def test_export_ao_elec_dip_deriv():
     PSI4_MU_Grad = mints.dipole_grad(D)
     G_python_MU_mat = psi4.core.Matrix.from_array(MU_Gradient)
     assert psi4.compare_matrices(PSI4_MU_Grad, G_python_MU_mat, 10, "DIPOLE_GRADIENT_TEST")  # TEST
+
+def test_export_ao_overlap_half_deriv():
+    h2o = psi4.geometry("""
+        O
+        H 1 1.0
+        H 1 1.0 2 101.5
+        symmetry c1
+    """)
+
+    psi4.core.set_active_molecule(h2o)
+    psi4.set_options({'basis': 'STO-3G'})
+
+    rhf_e, wfn = psi4.energy('SCF', return_wfn=True)
```

Comment:
fyi, can do these three lines with `rhf_e, wfn = psi4.energy('scf/sto-3g')`. or can add `molecule=h2o` to energy call.